### DISCUSSION TITLE
NX Release: Support Github Enterprise Slugs for Github Releases

### DISCUSSION THREAD

**Discussion by zachlewton** [nrwl/nx#22909] [Feature Requests]
Hello, i have been trying to build a github actions workflow with nx release but im having trouble getting the automated github release to work with a github enterprise url. i get the error  NX   Unable to create a GitHub release because the GitHub repo slug could not be determined. . digging through the source code it looks like the repo to post the release to is sourced from this function which would match an enterprise url https://github.com/nrwl/nx/blob/4d6cd36f5c222012265d474f1026cf8e341ffabe/packages/nx/src/command-line/release/utils/github.ts#L38

It would be great if a user could explicitly pass the github enterprise url either in the cli or in the release config in the nx.json

**Comment by mattridley** [comment]
Looking at this code it should be fairly straightforward to extract the enterprise url at the same time as the slug and use that throughout

**Reply by zachlewton** [reply]
closed this cause there is already an issue open but i dont think there is any movement on it

https://github.com/nrwl/nx/issues/21906
